src/test/java/com/commercetools/sync/inventory/InventorySyncTest.java

```diff
@@ -0,0 +1,214 @@
+                InventoryEntryMock.of(SKU_1, QUANTITY_1, RESTOCKABLE_1, DATE_1).withChannelRefExpanded(REF_2, KEY_2).build(),
+                InventoryEntryMock.of(SKU_2, QUANTITY_1, RESTOCKABLE_1, DATE_1).build(),
+                InventoryEntryMock.of(SKU_2, QUANTITY_1, RESTOCKABLE_1, DATE_1).withChannelRefExpanded(REF_1, KEY_1).build(),
+                InventoryEntryMock.of(SKU_2, QUANTITY_1, RESTOCKABLE_1, DATE_1).withChannelRefExpanded(REF_2, KEY_2).build()
+        );
+
+        drafts = asList(
+                InventoryEntryDraft.of(SKU_1, QUANTITY_1, DATE_1, RESTOCKABLE_1,null),
+                InventoryEntryDraft.of(SKU_1, QUANTITY_1, DATE_1, RESTOCKABLE_1, reference1),
+                InventoryEntryDraft.of(SKU_1, QUANTITY_1, DATE_1, RESTOCKABLE_1, Channel.referenceOfId(KEY_2)),
+                InventoryEntryDraft.of(SKU_2, QUANTITY_2, DATE_2, RESTOCKABLE_2,null),
+                InventoryEntryDraft.of(SKU_2, QUANTITY_2, DATE_2, RESTOCKABLE_2, reference1),
+                InventoryEntryDraft.of(SKU_2, QUANTITY_2, DATE_2, RESTOCKABLE_2, Channel.referenceOfId(KEY_2)),
+                InventoryEntryDraft.of(SKU_3, QUANTITY_1, DATE_1, RESTOCKABLE_1,null),
+                InventoryEntryDraft.of(SKU_3, QUANTITY_1, DATE_1, RESTOCKABLE_1, reference1),
+                InventoryEntryDraft.of(SKU_3, QUANTITY_1, DATE_1, RESTOCKABLE_1, Channel.referenceOfId(KEY_2))
+        );
+    }
+
+
+    @Test
+    public void syncDrafts_ShouldEndWithoutExceptions() {
+        getInventorySyncer(30, 1, false).syncDrafts(drafts);
+    }
+
+    @Test
+    public void syncDrafts_WithParallelOption_ShouldEndWithoutExceptions() {
+        final List<InventoryEntryDraft> moreDrafts = Stream.generate(() -> drafts)
+                .limit(20)
+                .flatMap(list -> list.stream())
+                .collect(Collectors.toList());
+        getInventorySyncer(10, 4, false).syncDrafts(moreDrafts);
+    }
+
+    @Test
+    public void getStatistics_ShouldReturnProperValues() {
+        final InventorySync inventorySync = getInventorySyncer(30, 1, false);
+        inventorySync.syncDrafts(drafts);
+        final InventorySyncStatistics stats = inventorySync.getStatistics();
+        assertThat(stats).isNotNull();
+        assertThat(stats.getProcessed()).isEqualTo(9);
+        assertThat(stats.getFailed()).isEqualTo(0);
+        assertThat(stats.getCreated()).isEqualTo(3);
+        assertThat(stats.getUpdated()).isEqualTo(3);
+    }
+
+    @Test
+    public void syncDrafts_WithEnsuredChannels_ShouldCreateEntriesWithUnknownChannels() {
+        final InventoryEntryDraft draftWithNewChannel = InventoryEntryDraft.of(SKU_3, QUANTITY_1, DATE_1, RESTOCKABLE_1,
+                Channel.referenceOfId(KEY_3));
+        final InventorySync inventorySync = getInventorySyncer(30, 1, true);
+        inventorySync.syncDrafts(asList(draftWithNewChannel));
```

Comment:
`asList()`: `Collections.singletonList()` is more expected here (i know, this is a result of JDK bad design, but now we should admit this and live with it :) )